packages/opentelemetry-plugin-mongodb-core/tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "outDir": "build"
   },
   "include": [
-    "src/**/*.ts",
-    "test/**/*.ts"
+    "src/**/*.ts"
```

Comment:
all `tsconfig.ts` has "test" too, what's the reason of removing it ?

Reply:
I've got a typescript issue when compiling:
```
Argument of type 'SimpleSpanProcessor' is not assignable to parameter of type 'SpanProcessor'.
  Types of property 'onStart' are incompatible.
    Type '(span: import("/home/vmarchaud/opentelemetry/opentelemetry-js/packages/opentelemetry-plugin-mongodb-core/node_modules/@opentelemetry/tracing/build/src/Span").Span) => void' is not assignable to type '(span: import("/home/vmarchaud/opentelemetry/opentelemetry-js/packages/opentelemetry-types/build/src/trace/span").Span) => void'.
      Types of parameters 'span' and 'span' are incompatible.
        Type 'Span' is missing the following properties from type 'Span': _tracer, spanContext, kind, attributes, and 17 more.ts(2345)
```
Which makes no sense since i had the same setup as other plugins then i found out that compiling the test aren't useful since we run them using `ts-mocha` which was working, i believe we could remove that from all other packages too.

Reply:
Merging PR now, please open an issue to discuss this separately.